pkg/relic/Makefile

```diff
@@ -10,14 +10,18 @@ endif
 .PHONY: all
 
 all: $(PKG_BUILDDIR)/Makefile
-	"$(MAKE)" -C $(PKG_BUILDDIR) && \
-	cp $(PKG_BUILDDIR)/lib/librelic_s.a $(BINDIR)$(PKG_NAME).a
+	@echo "Building relic package..." ; \
+		OUTPUT="$$("$(MAKE)" -C $(PKG_BUILDDIR) 2>&1)" || \
+		{ echo $$OUTPUT; false; } && \
```

Comment:
> You could also just chain it like: ```echo $$OUTPUT; cp ...``` instead of tampering with the return code for && to apply.

Could you write this out? I *don't* want to cp when the build failed, so ```echo $$output ; cp ...``` won't work. Also, after echo, the recipe still needs to fail.